java/arcs/jvm/storage/database/testutil/MockDatabaseManager.kt

```diff
@@ -114,7 +121,14 @@ open class MockDatabase : Database {
             Unit
         }
 
-    override suspend fun getAllStorageKeys() = data.keys
+    override suspend fun getAllStorageKeys(): Map<StorageKey, Type> {
+        val type = EntityType(Schema(
+            listOf<SchemaName>(),
+            SchemaFields(emptyMap(), emptyMap()),
+            ""
+        ))
+        return data.keys.map { it to CollectionType(type) }.toMap()
```

Comment:
Is every key guaranteed to be for a collection?

Reply:
updated the code to look at the type.